server/trafficrules/src/main/java/com/coffeejug/trafficrules/db/Progress.java

```diff
@@ -0,0 +1,51 @@
+package com.coffeejug.trafficrules.db;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+
+@Entity
+public class Progress {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
+
+    private String mockProgress;

```

Comment:
What is the purpose of `mockProgress`?

Reply:
What do u propose instead of ?

Reply:
I don't know what is its purpose :) The `mock` world in the production code looks weird to me

Reply:
So, seems it will live here until we invite some better structure for progress.

Reply:
I don't what structure you refer to but why isn't this field called `progress` then?